packages/split-layout/src/styles/vaadin-split-layout-base-styles.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @license
+ * Copyright (c) 2016 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import '@vaadin/component-base/src/style-props.js';
+import { css } from 'lit';
+
+export const splitLayoutStyles = css`
+  :host {
+    display: flex;
+    overflow: hidden !important;
+    transform: translateZ(0);
+  }
+
+  :host([hidden]) {
+    display: none !important;
+  }
+
+  :host([orientation='vertical']) {
+    flex-direction: column;
+  }
+
+  :host ::slotted(*) {
+    flex: 1 1 auto;
+    overflow: auto;
```

Comment:
This could be considered unnecessary or even unexpected. It also means that content is always clipped. Easy enough to work around, but like I said, maybe unexpected that we override this on the content element.

Reply:
The `overflow: auto` was added in a very early version: https://github.com/vaadin/vaadin-split-layout/commit/375735a4273631ba62777196c6c4cdf461d7c814.
We can probably indeed remove it from base styles, it probably shouldn't break anything.

Reply:
Removed.